package.json

```diff
@@ -11,9 +11,11 @@
     "lint-ci": "eslint src/",
     "test": "impunity --entry-point src/platform/web/main.js src/platform/web/Platform.js --force-esm-dirs lib/ src/ --root-dir src/",
     "test:postcss": "impunity --entry-point  scripts/postcss/tests/css-compile-variables.test.js scripts/postcss/tests/css-url-to-variables.test.js",
+    "test:sdk": "yarn build:sdk && cd ./scripts/sdk/test/ && yarn --no-lockfile && node test-sdk-in-esm-vite-build-env.js && node test-sdk-in-commonjs-env.js",
```

Comment:
This one is a bit clunky but it makes it easy to test whether `hydrogen-view-sdk` actually works in an ESM Vite build and when using `require` in Node.js (CommonJS).

All of the "testing" code was added in https://github.com/vector-im/hydrogen-web/pull/693/commits/2401b7f453a417dc87635e43aa2339ed2517600c if you want to review it on its own. Feel free to prompt to remove or different way to test.

Thought the testing was slightly needed as testing manually in each environment is cumbersome. And we really don't know whether the SDK actually works before pushing it out. All of the usage code is just documented in `SDK.md`